qemu/tests/mq_enabled_chk_windows.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_test
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    check netkvm multi-queue support is enabled - windows only
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    timeout = int(params.get("login_timeout", 360))
+    session = vm.wait_for_login(timeout=timeout)
+
+    # verify driver
+    error_context.context("Check if the driver is installed and "
+                          "verified", logging.info)
+    driver_name = params.get("driver_name", "netkvm")
+    session = utils_test.qemu.windrv_check_running_verifier(session, vm,
+                                                            test,
+                                                            driver_name,
+                                                            timeout)
+
+    msis, queues = utils_net.get_MSIs_and_queues_windows(params, vm)
+    if not msis or not queues:
+        test.fail("Can't get msi & queues info")
+    nic = vm.virtnet[0]
+    nic_queues = int(nic['queues'])
+    nic_vectors = int(nic['vectors']) if nic['vectors'] else (2 * queues + 2)
+    if nic_vectors != 2 * nic_queues + 2:
+        # custom vectors
+        nic_queues = (nic_vectors - 2) // 2
+        if nic_queues == 0:
+            nic_queues = 1

```

Comment:
The check looks very similar with this one: https://github.com/autotest/tp-qemu/commit/4200af4be92ed23fd7f17394f0d83476791fac4e#diff-fbb86110c111aabbe176bb0ea72ef488R84 Just the configuration of queues are different, so I suppose this test can reuse the previous script just with different cfg configuration.